dockers/docker-router-advertiser/docker-router-advertiser.supervisord.conf

```diff
@@ -3,28 +3,39 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
+[eventlistener:dependent-startup]
+command=python -m supervisord_dependent_startup
+autostart=true
+autorestart=unexpected
+startretries=0
+exitcodes=0,3
+events=PROCESS_STATE
+
 [eventlistener:supervisor-proc-exit-script]
 command=/usr/bin/supervisor-proc-exit-listener --container-name radv
 events=PROCESS_STATE_EXITED
 autostart=true
 autorestart=unexpected
 
-[program:start.sh]
-command=/usr/bin/start.sh
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n -iNONE
 priority=1
-autostart=true
+autostart=false
 autorestart=false
-startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
+dependent_startup=true
 
-[program:rsyslogd]
-command=/usr/sbin/rsyslogd -n
+[program:start.sh]
+command=/usr/bin/start.sh
 priority=2
 autostart=false
 autorestart=false
+startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
+dependent_startup=true
+dependent_startup_wait_for=rsyslogd:running
 
 [program:radvd]
```

Comment:
wonder why radvd does not have dependent_startup directive?

Reply:
move into j2 template.